modules/node/package.json

```diff
@@ -44,6 +44,7 @@
     "uuid": "3.3.3"
   },
   "devDependencies": {
+    "@connext/client": "1.4.1",
```

Comment:
If we're doing integration tests in `modules/integration-tests` then we should prob keep `module/node` unit tests focused on the node units rather than node-client interactions. Seems like this import could be removed eventually